src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/card/Card.kt

```diff
@@ -0,0 +1,34 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.card
+
+import javafx.scene.Node
+import javafx.scene.layout.VBox
+import tornadofx.*
+
+class Card: VBox() {
+
+    var cardChildren = observableList<Node>()
+
+
+    fun cardfront(init: CardFront.() -> Unit = {}): CardFront {
+        val cf = CardFront()
+        cf.init()
+        cardChildren.add(cf)
+        return cf
+    }
+
+    // todo    private val cardBack: CardBack by singleAssign()
```

Comment:
//TODO: I think is the right way to do this?

Basically, there's an "official" TODO syntax that gets tracked in intellij